pkg/migrate/Exec.go

```diff
@@ -0,0 +1,223 @@
+package migrate
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"time"
+)
+
+import (
+	"github.com/gobuffalo/pop"
+	"github.com/pkg/errors"
+)
+
+func Exec(inputReader io.Reader, tx *pop.Connection, wait time.Duration) error {
```

Comment:
Add a comment here about what this does.

```suggestion
// Exec executes ...
func Exec(inputReader io.Reader, tx *pop.Connection, wait time.Duration) error {
```